configs/server.toml

```diff
@@ -1,122 +1,337 @@
 [quic.certificate]
+# Indicates whether the QUIC certificate is self-signed.
+# `true` for self-signed certificates, often used in internal or testing environments.
+# `false` for certificates issued by a certificate authority, common in production.
 self_signed = true
+
+# Path to the QUIC TLS certificate file.
 cert_file = "certs/iggy_cert.pem"
+
+# Path to the QUIC TLS key file.
 key_file = "certs/iggy_key.pem"
 
+# Message cleaner configuration.
 [message_cleaner]
+# Enables or disables the background process for deleting expired messages.
+# `true` activates the message cleaner.
+# `false` turns it off, messages will not be auto-deleted based on expiry.
 enabled = true
+
+# Interval in seconds for running the message cleaner.
 interval = 60
 
+# Message saver configuration.
 [message_saver]
+# Enables or disables the background process for saving buffered data to disk.
+# `true` ensures data is periodically written to disk.
+# `false` turns off automatic saving, relying on other triggers for data persistence.
 enabled = true
+
+# Controls whether data saving is synchronous (enforce fsync) or asynchronous.
+# `true` for synchronous saving, ensuring data integrity at the cost of performance.
+# `false` for asynchronous saving, improving performance but with delayed data writing.
 enforce_fsync = true
+
+# Interval in seconds for running the message saver.
 interval = 30
 
+# Personal access token configuration.
 [personal_access_token]
+# Sets the maximum number of active tokens allowed per user.
 max_tokens_per_user = 100
 
+# Personal access token cleaner configuration.
 [personal_access_token.cleaner]
+# Enables or disables the token cleaner process.
+# `true` activates periodic token cleaning.
+# `false` disables it, tokens remain active until manually revoked or expired.
 enabled = true
+
+# Interval in seconds for running the token cleaner.
 interval = 60
 
+# System configuration.
 [system]
+# Base path for system data storage.
 path = "local_data"
 
+# Database configuration.
 [system.database]
+# Path for storing database files.
 path = "database"
 
+# Logging configuration.
 [system.logging]
+# Path for storing log files.
 path = "logs"
+
+# Level of logging detail. Options: "debug", "info", "warn", "error".
 level = "info"
+
+# Maximum size of log files in megabytes before rotation.
 max_size_megabytes = 512
+
+# Number of days to retain log files before deletion.
 retention_days = 7
 
+# Cache configuration.
 [system.cache]
+# Enables or disables the system cache.
+# `true` activates caching for frequently accessed data.
+# `false` disables caching, data is always read from the source.
 enabled = true
+
+# Maximum size of the cache, e.g., "4GB".
 size = "4GB"
 
+# Data retention policy configuration.
 [system.retention_policy]
+# Configures the message expiry setting.
+# "disabled" means messages are kept indefinitely.
+# A time value in seconds determines the lifespan of messages.
 message_expiry = "disabled"
```

Comment:
please mention that example format, eg. 2 days 4 hours 15 minutes

Reply:
Updated, please check if this example is good enough

Reply:
I saw this one https://github.com/iggy-rs/iggy/issues/277 mentioning about human readable values for configs. Would you like to already introduce it in this PR as an example? If yes, I can update it to the example proposed by you

Reply:
@jakkaz we're using now a human-friendly strings for parsing the time (docs site needs to be updated with this info too).

Reply:
gotcha, updated then to the human readable format together with example